tests/integration/config/plugins/integration-tests/data/tests/tests_mission_ships.txt

```diff
@@ -0,0 +1,217 @@
+test-data "One Earthly Barge"
+	category "savegame"
+	contents
+		pilot Bobbi Bughunter
+		date 16 11 3013
+		system Sol
+		planet Earth
+		ship "Star Barge"
+			name "Buggy Barge"
+			sprite "ship/star barge"
+			attributes
+				category "Light Freighter"
+				cost 190000
+				mass 70
+				bunks 3
+				"cargo space" 50
+				drag 2.1
+				"engine capacity" 40
+				"fuel capacity" 300
+				"heat dissipation" 0.8
+				hull 1000
+				"outfit space" 130
+				"required crew" 1
+				shields 600
+				"turret mounts" 1
+				"weapon capacity" 20
+			outfits
+				"X1700 Ion Thruster"
+				"X1200 Ion Steering"
+				Hyperdrive
+				"Anti-Missile Turret"
+				"D14-RN Shield Generator"
+				"nGVF-BB Fuel Cell"
+				"LP036a Battery Pack"
+			crew 1
+			fuel 300
+			shields 600
+			hull 1000
+			engine -9 38 1
+			engine 9 38 1
+			turret 0 -8 "Anti-Missile Turret"
+			leak leak 60 50
+			explode "tiny explosion" 10
+			explode "small explosion" 10
+			system Sol
+			planet Earth
+		account
+			credits 0
+			score 400
+			history
+		mission "Switch"
+			landing
+			invisible
+			on offer
+				take ship "Star Barge"
+				give ship "Sparrow" "The Sparrow"
+					id "ship: sparrow"
+		mission "Check sparrow"
+			invisible
+			on offer
+				take "Sparrow" "The sparrow"
```

Comment:
This is wrong.

Reply:
Oh, I'll look tomorrow
~crap he found my ruse to make green lights on the tests to reassure ppl~

Reply:
```suggestion
				take "Sparrow" "The Sparrow"
```